### ISSUE TITLE
[gui] window doesn't close and command prompt not commng back 

### ISSUE THREAD

**Issue by lslada** [internet-sicherheit/ethereum-cache-creator#36]
When running the  current artifact (https://github.com/internet-sicherheit/ethereum-cache-creator/actions/runs/116765443), with the gui param, the window doesn't close after filling the params, and pressing "create json file", and the command prompt never comes back when the program finishes. However, the output file is created normally.

**Comment by moekappels** [comment]
yeah that s simply a missing feature (the button "create json file" doesnt give any visual feedback because i havent programmed any yet). As far as i know the gui is not the focus of our attention right now, since the program is supposed to run on the server anyway. so i think we need clarification on whether or not the gui will remain relevant going foward.

**Comment by lslada** [comment]
OK, I was just reporting what I was testing. If the gui is a feature that will be interesting in the future, I have no problem in being responsible for it.

**Comment by kiview** [comment]
I don't see any near future plans for it, definitely not for extracting the data. Maybe more something in the direction of visualization (although it seems, there it might wiser to work with the extracted data with other data science tools, such as R or Python).

**Comment by moekappels** [comment]
is this still relevant?
